### DISCUSSION TITLE
Deleting Deta Micro

### DISCUSSION THREAD

**Discussion by BenitzCoding** [deta/cloud-docs#376] [Help]
# What am I trying to do?
I'm trying to delete my micro to re-create it into a project group thing, I want to delete and re-create the micro into the project group or move it in if that's possible.

**Comment by lemonyte** [comment]
Theres no way to move a micro between projects at the moment. You can delete the micro in the settings for it on the Deta web interface, and then create it again under a different project with the command `deta new --project your_project --name micro_name` (with any other flags as needed).

**Comment by abdelhai** [comment]
you can delete a micro from it's settings page. then, locally, delete the `.deta` directory then run `deta new` inside the project's directory.

**Comment by felipebastosweb** [comment]
I deleted the micro in the settings for it on the Deta web interface, the micro record was erased, but it keeps running https://h0wfx5.deta.dev/api/categories  https://oxrhik.deta.dev/ .. I would like to completely turn off the micro even to stop costs for Deta.

**Reply by jnsougata** [reply]
> I deleted the micro in the settings for it on the Deta web interface, the micro record was erased, but it keeps running https://h0wfx5.deta.dev/api/categories https://oxrhik.deta.dev/ .. I would like to completely turn off the micro even to stop costs for Deta.

see it's gone now! just give it a little bit of time
